examples/aishell/NST/run_nst.sh

```diff
@@ -0,0 +1,434 @@
+  echo "********stage 3 start time : $now ********"
+  python split_data_list.py \
+    --job_nums $num_split \
+    --data_list_path data/train/$data_list \
+    --output_dir data/train/$dir_split
+
+fi
+
+
+# stage 4 will perform inference without language model on the given sublist(job num)
+# here is example usages:
+# bash run_nst.sh --stage 4 --stop-stage 4 --job_num $i --dir_split data/train/wenet_4khr_split_60/
+# --hypo_name hypothesis_0.txt --dir exp/conformer_aishell2_wenet4k_nst4
+# You need to specify the "job_num" n (n <= N), "dir_split" which is the dir path for split data
+# "hypo_name" is the path for output hypothesis and "dir" is the path where we train and store the model.
+# For each gpu, you can run with different job_num to perform data-wise parallel computing.
+if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
+  echo "********stage 4 start time : $now ********"
+  # we assume you have run stage 2 so that avg_${average_num}.pt exists
+  decode_checkpoint=$dir/avg_${average_num}.pt
+  # Please specify decoding_chunk_size for unified streaming and
+  # non-streaming model. The default value is -1, which is full chunk
+  # for non-streaming inference.
+  decoding_chunk_size=
+  ctc_weight=0.5
+  reverse_weight=0.0
+  mode="attention_rescoring"
+  gpu_id=0
+  echo "job number  ${job_num} "
+  echo "data_list dir is  ${dir_split}"
+  echo "hypo name is " $hypo_name
+  echo "dir is ${dir}"
+
+  python wenet/bin/recognize.py --gpu $gpu_id \
+    --mode $mode \
+    --config $dir/train.yaml \
+    --data_type $data_type \
+    --test_data data/train/${dir_split}data_sublist${job_num}/data_list \
+    --checkpoint $decode_checkpoint \
+    --beam_size 10 \
+    --batch_size 1 \
+    --penalty 0.0 \
+    --dict $dict \
+    --ctc_weight $ctc_weight \
+    --reverse_weight $reverse_weight \
+    --result_file data/train/${dir_split}data_sublist${job_num}/${hypo_name} \
+    ${decoding_chunk_size:+--decoding_chunk_size $decoding_chunk_size}
+    echo "end time : $now"
+
+fi
+
+
+# Generate wav.scp file and label.txt file(optional) for each sublist we generated in step 3.
+# the wav_dir should be prepared in data processing step as we mentioned.
+#You need to specify the "job_num" n (n <= N), "dir_split" which is the dir path for split data,
+# "hypo_name" is the path for output hypothesis and "dir" is the path where we train and store the model.
+# wav_dir is the directory that stores raw wav file and possible labels.
+# if you have label for unsupervised dataset, set label = 1 other wise keep it 0
+# For each gpu or cpu, you can run with different job_num to perform data-wise parallel computing.
+if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ] && [ ${enable_nst} -eq 0 ]; then
+  echo "********stage 5 start time : $now ********"
+  python get_wav_labels.py \
+    --dir_split data/train/${dir_split} \
+    --hypo_name /$hypo_name \
+    --wav_dir $wav_dir\
+    --job_num $job_num \
+    --label $label
```

Comment:
`PYTHONPATH` in `path.sh` does not include $PWD/local, I this line 338 should be
```py
python local/get_wav_labels.py
```